### PR TITLE
Add suffixes to typos from "ace" to "ad"

### PR DESCRIPTION
**Pull request by int-y1**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -1115,9 +1115,24 @@ acdept->accept, adept,
...
+acedemia->academia
 acedemic->academic
 acedemically->academically
+acedemics->academics
+acedemy->academy
+acelerate->accelerate
 acelerated->accelerated
+acelerates->accelerates
+acelerating->accelerating
+aceleration->acceleration
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:1131]: Perhaps add plural:
```
acelerations->accelerations
```

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -1115,9 +1115,24 @@ acdept->accept, adept,
 acditionally->additionally
 acecess->access
 acedamia->academia
+acedamic->academic
+acedamically->academically
+acedamics->academics
+acedamy->academy
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:1121]: Perhaps add plural:
```
acedamies->academies
```

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -1115,9 +1115,24 @@ acdept->accept, adept,
...
+acedemia->academia
 acedemic->academic
 acedemically->academically
+acedemics->academics
+acedemy->academy
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:1126]: Again plural:
```
acedemies->academies
```

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -1578,6 +1656,10 @@ adapive->adaptive
 adaptacion->adaptation
 adaptaion->adaptation
 adaptare->adapter
+adaptate->adapt
+adaptated->adapted
+adaptater->adapter
+adaptaters->adapters
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:1662]: This seems to be missing:
```
adaptates->adapts
```

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -1685,6 +1782,17 @@ addjust->adjust
...
 addjusting->adjusting
 addjusts->adjusts
+addmin->admin
+addminister->administer
+addministered->administered
+addministering->administering
+addministers->administers
+addministration->administration
+addministrations->administrations
+addministrative->administrative
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:1792]:
```
addministratively->administratively
```

File: codespell_lib/data/dictionary.txt [hunk 6]
```diff
@@ -1819,24 +1940,60 @@ adjustmants->adjustments
 adjustmenet->adjustment
 admendment->amendment
 admi->admin
+admininister->administer
+admininistered->administered
+admininistering->administering
+admininisters->administers
+admininistration->administration
+admininistrations->administrations
 admininistrative->administrative
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:1949]:
```
admininistratively->administratively
```

File: codespell_lib/data/dictionary.txt [hunk 7]
```diff
@@ -1819,24 +1940,60 @@ adjustmants->adjustments
...
 admininistrator->administrator
 admininistrators->administrators
+admininster->administer
+admininstered->administered
+admininstering->administering
+admininsters->administers
+admininstration->administration
+admininstrations->administrations
+admininstrative->administrative
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:1958]:
```
admininstratively->administratively
```

File: codespell_lib/data/dictionary.txt [hunk 8]
```diff
@@ -1819,24 +1940,60 @@ adjustmants->adjustments
...
 administraively->administratively
+administraor->administrator
+administraors->administrators
+adminiter->administer
+adminitered->administered
+adminitering->administering
+adminiters->administers
+adminitration->administration
+adminitrations->administrations
+adminitrative->administrative
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:1980]:
```
adminitratively->administratively
```

File: codespell_lib/data/dictionary.txt [hunk 9]
```diff
@@ -1819,24 +1940,60 @@ adjustmants->adjustments
...
+adminitrators->administrators
 adminssion->admission
+adminster->administer
 adminstered->administered
+adminstering->administering
+adminsters->administers
 adminstrate->administrate
 adminstration->administration
+adminstrations->administrations
 adminstrative->administrative
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:1991]:
```
adminstratively->administratively
```

File: codespell_lib/data/dictionary.txt [hunk 10]
```diff
@@ -1871,6 +2035,8 @@ adrersses->addresses
 adrerssing->addressing
 adres->address
 adresable->addressable
+adresed->addressed
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:2038]: Perhaps:
```
adresee->addressee
adresees->addressees
```

Comment (int-y1) [reply codespell_lib/data/dictionary.txt:2038]: I also added `adresees->addresses`

**Review by DimitriPapadopoulos — APPROVED** [review]

**Pull request merged**